components/ReactBKKLogo/RevealEllipse.js

```diff
@@ -44,6 +45,17 @@ class RevealEllipse extends React.Component {
       this.timer = Date.now()
       this.raf = window.requestAnimationFrame(this.revealStep)
     }, this.props.delay)
+    document.addEventListener('click', function (e) {
```

Comment:
Again, this is not the right place to put this code, because if the RevealEllipse is removed from the page, smooth scrolling will stop working.

Ideally, it should be put in `onClick` listener at the root element.